spec/lib/phobos/producer_spec.rb

```diff
@@ -224,9 +274,10 @@ class TestProducer1
 
     describe 'without a kafka_client configured' do
       it 'creates a new client and an async_producer bound to the current thread' do
+        config = Phobos.config.producer_hash.reject { |k, _| %i(persistent_connections).include?(k) }
```

Comment:
Are you able to use the constant `INTERNAL_PRODUCER_PARAMS` here?

Reply:
yep, will fix